### ISSUE TITLE
Installing with kingfisher.yaml leads to aria2c issue + fix

### ISSUE THREAD

**Issue by 1onic** [wwood/kingfisher-download#14]
When installing w/ provided env yaml, leads to the following trace:

```
12/16/2021 03:54:06 AM INFO: Kingfisher v0.0.1-dev
12/16/2021 03:54:06 AM INFO: Attempting download method aws-http for run SRR5036382 ..
12/16/2021 03:54:07 AM INFO: Found ODP link https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR5036382/SRR5036382
12/16/2021 03:54:07 AM INFO: Downloading .SRA file from AWS Open Data Program HTTP link using aria2c ..
aria2c: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```

then running:
```
ldd $(which aria2c) 
        linux-vdso.so.1 =>  (0x00007ffff3bdf000)
        libz.so.1 => .../kingfisher/bin/../lib/libz.so.1 (0x00007fd57e252000)
        libxml2.so.2 => .../kingfisher/bin/../lib/libxml2.so.2 (0x00007fd57e0e9000)
        libsqlite3.so.0 => .../kingfisher/bin/../lib/libsqlite3.so.0 (0x00007fd57df1d000)
        libssl.so.1.0.0 => not found
        libcrypto.so.1.0.0 => not found
        libssh2.so.1 => .../kingfisher/bin/../lib/libssh2.so.1 (0x00007fd57e08b000)
        libcares.so.2 => .../kingfisher/bin/../lib/libcares.so.2 (0x00007fd57e071000)
        libstdc++.so.6 => .../kingfisher/bin/../lib/libstdc++.so.6 (0x00007fd57dd72000)
```

the following yaml/conda correction lead to a solution in my case, since openssl==3.0 using kingfisher.yaml:
`conda install -c bioconda samtools openssl=1.0`

Posting in case any else has this issue come up.

**Comment by wwood** [comment]
Thanks. My guess as to what is happening is that you installed into a non-empty environment, and somehow it was choosing an out of date aria2c conda to install. 

I just pushed a patch which I think will address that issue. I'm not sure that you can test it out now though?

**Comment by 1onic** [comment]
It was installed into an empty env with  "-f kingfisher.yaml". So I am not sure if the aria2c conda install was the issue. I think the openssl=3.0 was the issue since openssl=1.0 fixed it.

**Comment by wwood** [comment]
OK. I guess is that solved by the patch anyway. My memory is that older aria2c versions had this problem. The newest version I submitted to conda-forge and it was fixed there, and I guess you were installing an older version because that is what conda chose, through no fault of your own. 

We can be reasonably certain it isn't a general error because the GitHub actions run the tests using a newly recreated conda env built from kingfisher.yml each time. Still, thanks for reporting it.
